scipy/sparse/_coo.py

```diff
@@ -732,13 +735,18 @@ class coo_matrix(spmatrix, _coo_base):
     ndim : int
         Number of dimensions (this is always 2)
     nnz
```

Comment:
Shouldn't all be documented? E.g.:

```suggestion
    nnz
        Number of stored values, including explicit zeros
```

Reply:
@ivirshup Can you confirm if the docstrings for these come from the properties themselves?

Reply:
The rendered docs suggest that the properties pull those doc_strings into the class doc_string. 
It doesn't help the readability of the code, but the online docs render well. :)